pkgs/development/tools/coder/default.nix

```
@@ -42,29 +72,43 @@ buildGoModule rec {
   subPackages = [ "cmd/..." ];
 
   preBuild = ''
-    export HOME=$TEMPDIR
+    export HOME=$(mktemp -d)
+
```

From testing locally, this allows us to pick up the ESBuild binary

```suggestion
    export HOME=$(mktemp -d)
    export ESBUILD_BINARY_PATH=${ESBUILD_BINARY_PATH}
```